chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -1,21 +1,20 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
-<%
-baseUrl = request.registry.settings['api_url']
-%>
+  <%
+    baseUrl = request.registry.settings['api_url']
+  %>
   <iframe src="${baseUrl}/rest/services/all/MapServer/${c['layerBodId']}/${c['featureId']}/iframeHtmlPopup?lang=${lang}" width="100%" height="165" frameborder="0" style="border: 0;" scrolling="no"></iframe>
 </%def>
 
-
 <%def name="iframe_content(c, lang)">
 <%
     from pyramid.url import route_url
     protocol = request.scheme
     lang = request.lang
     topic = request.matchdict.get('map')
     host = h.make_agnostic(request.host_url + request.uscript_name)
-    type_station = c['attributes']['betriebspunkttyp']
+    type_station = c['attributes']['betriebspunkttyp_de']
```

Comment:
I think you have two attributes here betriebspunkttyp_de and betriebspunkttyp_fr. Is thi intended to use only the de?

Reply:
Yes, only _de is used for defining the station type. _de and _fr are used for label display.